crates/cheatnet/src/runtime_extensions/forge_runtime_extension/mod.rs

```diff
@@ -508,6 +508,37 @@ impl<'a> ExtensionLogic for ForgeExtension<'a> {
                 }
                 Ok(CheatcodeHandlingResult::Handled(output))
             }
+            "print_call_entry_point" => {
+                let mut idx = 0;
+                let entry_point_type = match read_felt(&inputs, &mut idx).to_u8() {
+                    Some(0) => EntryPointType::Constructor,
+                    Some(1) => EntryPointType::External,
+                    Some(2) => EntryPointType::L1Handler,
+                    _ => unreachable!("Invalid EntryPointType variant"),
```

Comment:
This is bad, but I assume it will be fixed when we have a more generic solution (custom error being wrapped with an error message about version of snforge_std) in #1440

Reply:
Canno we return unrecoverable error from here?

Reply:
Not really since it is later cast to `HintError`, this error will result in `CairoRunError` and be evaluated to failed test result